engi_glfw.go

```diff
@@ -144,8 +137,8 @@ func run(defaultScene Scene, title string, width, height int, fullscreen bool) {
 				}
 
 				for _, s := range scene.world.Systems() {
-					if render, ok := s.(*RenderSystem); ok {
-						render.defaultBatch.SetProjection(gameWidth, gameHeight)
+					if _, ok := s.(*RenderSystem); ok {
+						Shaders.def.SetProjection(gameWidth, gameHeight)
 					}
```

Comment:
TODO: we shouldn't call `SetProjection` to those specific shaders..., at least not within a loop which just checks 'do we have a RenderSystem'.